curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/62017f47c87be96457c49f46.md

```diff
@@ -39,7 +39,8 @@ Your `span[class~="sr-only"]` selector should have the `-webkit-clip-path` prope
 
 ```js
 // this one gets removed apparently
-assert(code.includes('-webkit-clip-path: inset(50%) !important;'));
+const webkitRegex = /[\s]+-webkit-clip-path[\s]*:[\s]*inset\([\s]*50%[\s]*\) !important;/
```

Comment:
I don't understand why you're using `[\s]*` instead of just `\s*`.  When using a metacharacter like `\s` (match spaces), `\d` (match digits) or `\w` (match "word" characters (and underscore)), you don't need to open a character class. you only need the brackets if you're listing raw characters.

Reply:
It's an old hobby I'm trying to break.

Reply:
There are 4 more instances of these that should be removed as well.  Please take a moment to review your entire change before pushing up another commit. 

I also note that there is a space between the last parenthesis and the `!important`, which is also optional (according to the spec)

Reply:
Done